aspnetcore/tutorials/web-api-javascript.md

```diff
@@ -40,11 +40,13 @@ The simplest `fetch` call accepts a single parameter representing the route. A s
 
 1. Create a *wwwroot* directory in the project root.
 
+1. Create a *js* directory inside of the *wwwroot* directory.
+
 1. Add an HTML file named *index.html* to the *wwwroot* directory. Replace its contents with the following markup:
 
     [!code-html[](first-web-api/samples/3.0/TodoApi/wwwroot/index.html)]
 
-1. Add a JavaScript file named *site.js* to the *wwwroot* directory. Replace its contents with the following code:
+1. Add a JavaScript file named *site.js* to the *wwwroot/js* directory. Replace its contents with the following code:
```

Comment:
1. Add a JavaScript file named *site.js* to the *wwwroot/js* folder. Replace the contents of the *site.js* file with the following code: